### PR TITLE
LRQA-75809 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-75809

### CODE REVIEW

**Comment by Sharryxue** [comment]
ci:test:sf

**Comment by Sharryxue** [comment]
Just starting review.

File: portal-web/test/functional/com/liferay/portalweb/macros/AppManager.macro [hunk 1]
```diff
@@ -41,7 +41,9 @@ definition {
 			locator1 = "AppManager#USER_SIGN_IN",
 			value1 = "Sign In");
 
-		WaitForPageLoad();
+		while (IsElementNotPresent(key_text = "Email Address", locator1 = "AppManager#LOGIN_INPUT_FIELD")) {
```
Comment (Sharryxue) [inline portal-web/test/functional/com/liferay/portalweb/macros/AppManager.macro]: How about change this to use waitForElementPresent function?

**Review by Sharryxue — CHANGES_REQUESTED** [review]
Hello @melodywu,
Could you try if the waitForElementPresent function works for this? Thanks in advance.

File: portal-web/test/functional/com/liferay/portalweb/macros/AppManager.macro (hunk 1 above)
Comment (melodywu) [reply portal-web/test/functional/com/liferay/portalweb/macros/AppManager.macro]: done

**Comment by melodywu** [comment]
ci:test

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test

**Comment by melodywu** [comment]
ci:test

**Comment by melodywu** [comment]
ci:test:sf

**Comment by Sharryxue** [comment]
Just starting review.

**Comment by Sharryxue** [comment]
No changes needed. :)

**Comment by Sharryxue** [comment]
Rebase and resend.
https://github.com/Sharryxue/liferay-portal/pull/454

**Pull request closed without merging**